src/react/components/_dashboard/NotificationsCard/Tabs.module.scss

```diff
@@ -0,0 +1,33 @@
+@import 'src/react/stylesheets/variables';
+@import 'src/react/stylesheets/mixins';
+
+$height: 42px;
```

Comment:
Just asking simple question, @developish -- should this be put into `variables` and not hard-coded?

But maybe you have reason why not?

Reply:
I consider this to be a _local_ variable to this file. 42px high doesn't mean anything particular to put it in the global variables file